relay/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	// do not need to specify binlog-pos, because relay will fetch the whole file
 	BinLogName string `toml:"binlog-name" json:"binlog-name"`
 	BinlogGTID string `toml:"binlog-gtid" json:"binlog-gtid"`
+	UUIDSuffix string `toml:"-" json:"-"`
```

Comment:
maybe could change this variable to `int`, because in `ParseSuffixForUUID` we convert it to int and there's already `AddSuffixForUUID`